templates/uchiwa_config.json.j2

```diff
@@ -5,8 +5,10 @@
        "host": "{{ sensu_api_host }}",
        "ssl": {{ sensu_api_ssl | lower }},
        "port": {{ uchiwa_sensu_api_port }},
+       {% if sensu_api_user_name | length > 0 %}
```

Comment:
I don't _think_ you want to use the `| length >0` logic here as `sensu_api_user_name` should only be a string. I believe that if you don't define it, it'll evaluate to false allowing this to simply be:
```
{% if sensu_api_user_name %}
```
instead.